src/Game.cpp

```diff
@@ -9,24 +9,34 @@ Game::Game(QList<QChar> board, QList<quint8> freeCells)
 {
 }
 
+void Game::emptyBoard(void)
+{
+    m_freeCells.clear();
+    for(int i=0; i < m_board.size(); i++)
+    {
+        m_freeCells.append(i);
+        m_board[i] = ' ';
+    }
```

Comment:
This can be rewrite as
```suggestion
    m_board = { ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ' };
    m_freeCells = { 0, 1, 2, 3, 4, 5, 6, 7, 8 };
```

Reply:
This may be cleaner, but I think it won't work when the size of the board will be dynamic (eg 4x4 or 5x5). Don' you think?

Reply:
Yes, but for now all is hardcoded for 3x3, we need to do many hack in order to support dynamic grid :D